<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AmmunitionDialog</class>
 <widget class="QDialog" name="AmmunitionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>90</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Ammunition</string>
  </property>
  <widget class="QPushButton" name="applyButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>90</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="ammunitionMeasurementUnit">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>35</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>round(s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ammunitionInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
